CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
   - Removed sorting options by file size from `UploadFragment`.
 - Project:
   - Migrated Gradle builds from Groovy to Kotlin.
+- Widget:
+  - SocialApi doesn't use `GET /sources` method anymore
```

Comment:
```suggestion
  - SocialApi doesn't use `GET /sources` method anymore.
```

(Let's keep some consistency with dots).

Reply:
Also, let's order changes from the most important parts of the integration to the least: library, widget, project, example.